src/Common/Commands.ResourceManager.Profile/Tenant/GetAzureRMTenant.cs

```diff
@@ -0,0 +1,41 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.ResourceManager.Common;
+using Microsoft.Azure.Common.Authentication.Models;
+using System.Collections.Generic;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Profile
+{    
+    /// <summary>
+    /// Cmdlet to get user tenant information. 
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "AzureRMTenant")]
+    [Alias("Get-AzureRMDomain")]
+    [OutputType(typeof(List<AzureTenant>))]
```

Comment:
According to the task details - "If no tenant is specified, list all tenants.". So it can be a list.